source/wasm/src/lib.rs

```diff
@@ -672,21 +672,21 @@ pub fn generate_docs(additional_program: Option<ProgramConfig>) -> Vec<IDocFile>
 
 fn get_debugger_from_openqasm(
     sources: &[(Arc<str>, Arc<str>)],
-    capabilities: TargetCapabilityFlags,
+    profile: Profile,
 ) -> Result<(String, interpret::Interpreter), String> {
-    get_configured_interpreter_from_openqasm(sources, capabilities, true)
+    get_configured_interpreter_from_openqasm(sources, profile, true)
 }
 
 fn get_interpreter_from_openqasm(
     sources: &[(Arc<str>, Arc<str>)],
-    capabilities: TargetCapabilityFlags,
+    profile: Profile,
 ) -> Result<(String, interpret::Interpreter), String> {
-    get_configured_interpreter_from_openqasm(sources, capabilities, false)
+    get_configured_interpreter_from_openqasm(sources, profile, false)
 }
 
 fn get_configured_interpreter_from_openqasm(
     sources: &[(Arc<str>, Arc<str>)],
-    capabilities: TargetCapabilityFlags,
+    profile: Profile,
     dbg: bool,
 ) -> Result<(String, interpret::Interpreter), String> {
```

Comment:
Why the change to profile from capablilties?

Reply:
Oh then _can_ remain as capabilities. That should cut down the number of changes. Thanks!